src/Parser/Token.hs

```diff
@@ -151,6 +153,8 @@ instance Show TokenClass where
   show TcCBrace = "}"
   show TcOSquare = "["
   show TcCSquare = "]"
+  show TcOBanana = "(|"
+  show TcCBanana = "|)"
```

Comment:
Can you update `compiler/Errors.hs`, so that the bounds in `highlightAmulet` are correct? Sorry, I realise it's ugly.

Reply:
Of course